base/staticdata.jl

```diff
@@ -340,24 +340,22 @@ function verify_call(@nospecialize(sig), expecteds::Core.SimpleVector, i::Int, n
     return minworld[], maxworld[], result
 end
 
+const METHOD_ISMINMAX_INVOKE_LATEST = 0x1
+const METHOD_ISMINMAX_CALL_LATEST = 0x2
```

Comment:
which calls / queries is this quantifying over exactly?

If it's "any call w/ sig <: method.sig" then it seems that this bit just represents whether the Method is more-specific than all intersecting Methods. Is that the right intuition?